blocks/footer/footer.css

```diff
@@ -1,14 +1,301 @@
+  color: var(--text-color);
+  font-size: 22px;
   margin: 0;
-}
\ No newline at end of file
+  line-height: 20px;
+}
+
+.footer .top ul li {
+  flex: 1;
+}
+
+.footer .bottom .icon {
+  width: 35px;
+  height: 35px;
+}
+
+.footer .top .icon {
+  height: 50px;
+  width: 100%;
+  margin: 18px 5px 28px;
+}
+
+.footer .top h4 > .icon {
+  width: 270px;
+  height: 83px;
+}
+
+.footer .top em {
+  position: relative;
+  top: 119px;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.footer .top h4 > .icon,
+.footer .top ul li .icon {
+  filter: invert(65%) sepia(25%) saturate(321%) hue-rotate(13deg) brightness(97%) contrast(91%);
+  opacity: 0.8;
+}
+
+.footer .top > div:nth-of-type(2) {
+  overflow: scroll;
+  padding: 0 5px 40px;
+
+  /* Hide scrollbar for IE, Edge and Firefox */
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.footer .top > div:nth-of-type(2)::-webkit-scrollbar {
+  display: none;
+}
+
+.footer .top h4 > .icon:hover,
+.footer .top ul li .icon:hover {
+  filter: none;
+  opacity: 1;
+}
+
+.footer .top h4 > .icon svg,
+.footer .top ul li .icon svg {
+  filter: grayscale(1);
+}
+
+.footer .top h4 > .icon:hover svg,
+.footer .top ul li .icon:hover svg {
+  filter: none;
+}
+
+.footer .center {
+  background-color: var(--highlight-footer-background-color);
+  padding: 54px 15px 0;
+}
+
+.footer .bottom {
+  padding: 15px;
+}
+
+.footer .bottom, .footer .bottom a {
+  color: var(--breadcrumbs-color);
+}
+
+.footer .bottom ul li a {
+  margin: 0 18px;
+}
+
+.footer .bottom a:hover {
+  color: var(--link-hover-color-light);
+}
+
+.footer .center a:hover {
+  text-decoration: none;
+}
+
+.footer .bottom ul li:first-of-type a {
+  margin-left: 0;
+}
+
+.footer .bottom > div > div:first-of-type li a {
+  margin: 0 10px 0 0;
+}
+
+
+.footer .bottom > div > div:first-of-type > ul li:not(:first-of-type)::before {
+  content: '•';
+  font-size: 75%;
+  margin-right: 9px;
+}
+
+.footer .bottom > div > div > p {
+  font-size: var(--body-font-size-s);
+}
+
+.footer .center input {
+  background: var(--background-color);
+  border: 1px solid var(--heading-color);
+  box-sizing: border-box;
+  width: 100%;
+  padding: 23px 12px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: var(--body-font-size-m);
+  line-height: 18px;
+  color: var(--breadcrumbs-color);
+  margin: 15px 0;
+}
+
+.footer .center .button {
+  width: 100%;
+  margin-bottom: 25px;
+  margin-top: 15px;
+}
+
+.footer .center .button.disabled {
+  pointer-events: none;
+  cursor: default;
+  opacity: 0.5;
+}
+
+.footer .center ul {
+  justify-content: end;
+  font-size: var(--body-font-size-s);
+}
+
+.footer .center ul li ul li {
+  margin-bottom: 13px;
+}
+
+.footer .center ul li ul {
+  margin-top: 13px;
+  font-size: var(--body-font-size-xs);
+}
+
+.footer .top > div:first-of-type {
+  margin-bottom: 5px;
+  text-align: center;
+  border-bottom: 1px solid var(--highlight-footer-background-color);
```

Comment:
I usually don't name variables based on where they are used. If the same color is used elsewhere, it doesn't read well.